### PR TITLE
New CLI script to generate and verify MD5 hashes for entire cacti source

### PR DESCRIPTION
**Pull request by netniV**
This is a PHP version of #1279 which verifies the sources.  This can be doubly verified by running `md5sum -c < .md5sum`

The script itself runs in the reverse of md5sum as it verifies by default.  There are numerous exit codes that can be used from other scripts to identify failure should it need to be run in quiet mode.

### CODE REVIEW

**Comment by cigamit** [comment]
@netniV, please remove the .md5sum file from the pull request.  Rony can add that step to the packaging script.

**Comment by cigamit** [comment]
Same would apply for @paulgevers and @DavidLiedke and our EPEL packager.  Who is that again?

**Comment by netniV** [comment]
Should now be removed.

**Comment by DavidLiedke** [comment]
#349 can be closed now.

netniV thanks for converting.

File: cli/md5sum.php [hunk 1]
```diff
@@ -0,0 +1,369 @@
+#!/usr/bin/php -q
```
Comment (pautiina) [inline cli/md5sum.php]: I think it's better to use code that will work on BSD:
**#!/usr/bin/env php**

Comment (netniV) [reply cli/md5sum.php]: @pautiina I think the same script header is on the other PHP files. If you think we should change it for all of them, I reckon we should open that as a separate issue.

Comment (pautiina) [reply cli/md5sum.php]: It would be correct to do this for all files.

**Comment by cigamit** [comment]
Yes, and mark executable for all cli and poller* files.

File: cli/md5sum.php (hunk 1 above)
Comment (cigamit) [reply cli/md5sum.php]: Yes, and mark executable for all cli and poller* files.

**Comment by netniV** [comment]
I will work on moving some of the CLI functions I used in my md5sum script into the others and get them all operating consistently as well.  I'm thinking of a lib/cli.php to hold the common display_version, etc.

That way we can provide some basic functionality such as versioning/help/argument checking before we try and include the full global and go on to database access.  No point opening the database unless we need it.

**Pull request merged**